nncf/experimental/openvino_native/quantization/quantize.py

```diff
@@ -34,6 +34,24 @@
 from nncf.common.utils.timer import timer
 
 
+def dump_parameters(model: ov.Model, parameters: Dict, path: Optional[List] = []) -> None:
+    """
+    Dumps input parameters into Model's meta section.
+
+    :param model: ov.Model instance.
+    :param parameters: Incoming dictionary with parameters to save.
+    :param path: Optional list of the paths.
+    """
+    for key, value in parameters.items():
+        # Special condition for composed fields like IgnoredScope
+        if isinstance(value, IgnoredScope):
+            dump_parameters(model, value.__dict__, [key])
+            continue
+        value = value if value else None
+        rt_path = ['optimization'] + path + [key]
```

Comment:
I think "optimization" is quite an ambiguous name here. My take is that we should call it "nncf" and add another level internally and call it like "ptq" or "quantization" or whatever because we can come up with other post-training APIs at some point, e.g. "weight_compression" and we should also mentioned this in the rt info.